packages/client/hmi-client/src/page/data-explorer/components/tera-search-results-list.vue

```diff
@@ -99,9 +101,62 @@ const props = defineProps({
 	}
 });
 
-const previewedAsset = ref<ResultType | null>(null);
+const selectedAsset = ref();
 
-const emit = defineEmits(['toggle-data-item-selected']);
+const projectOptions = computed(() => [
+	{
+		label: 'Add this to...',
```

Comment:
```suggestion
		label: 'Select Project to add asset',
```

It will be nicer if `asset` was the type of the asset, but that's not vital.